Makefile

```diff
@@ -118,9 +118,9 @@ ifndef $(GOPATH)
 	export GOPATH
 endif
 
-ARGOEXEC_PKGS    := $(shell echo cmd/argoexec            && go list -f '{{ join .Deps "\n" }}' ./cmd/argoexec/            | grep 'argoproj/argo-workflows/v3/' | cut -c 39-)
-CLI_PKGS         := $(shell echo cmd/argo                && go list -f '{{ join .Deps "\n" }}' ./cmd/argo/                | grep 'argoproj/argo-workflows/v3/' | cut -c 39-)
-CONTROLLER_PKGS  := $(shell echo cmd/workflow-controller && go list -f '{{ join .Deps "\n" }}' ./cmd/workflow-controller/ | grep 'argoproj/argo-workflows/v3/' | cut -c 39-)
+ARGOEXEC_PKG_FILES    := $(shell echo cmd/argoexec            && go list -f '{{ join .Deps "\n" }}' ./cmd/argoexec/            | grep 'argoproj/argo-workflows/v3/' | cut -c 39- | xargs -I {} find {} -type f -name "*.go")
```

Comment:
a solution that uses more `go list` and less of other commands would be preferable since it would be more intelligent as well as more portable cross-platform, but this works on my Mac

for instance, I got this working:
```sh
go list -f '{{ join .Deps "\n" }}' ./cmd/argoexec/ |  grep 'argoproj/argo-workflows/v3/' | xargs go list -f '{{ range $file := .GoFiles }}{{ print $.ImportPath "/" $file "\n" }}{{ end }}' | cut -c 39-
```

this produces a shorter list -- it ignores test files and only uses the files of the package, not all files in the dir. I think it's more precise, but I might be wrong on that one and maybe I'm missing something

Reply:
Yes, I agree.

Reply:
The outputs are [above](https://github.com/argoproj/argo-workflows/pull/13145#discussion_r1632865882) for diffing. The original list output was not `uniq` so the number of lines of diff was higher than expected.